python-packages/contract_wrappers/test/test_erc20_wrapper.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for ERC20Token wrapper."""
+from decimal import Decimal
+import pytest
```

Comment:
add a blank line here, to separate standard library imports from 3rd party imports.

Reply:
done.